probcal/evaluation/kernels.py

```diff
@@ -2,6 +2,30 @@
 import torch
 
 
+def l2_norm(x: torch.Tensor, y: torch.Tensor) -> torch.Tensor:
```

Comment:
Let's add an underscore or something to the name to show that this isn't a kernel itself to be imported. So `_l2_norm`

Reply:
I am going to remove this entirely as it was a quick experiment towards mixed precision MCMD. The experiment failed, and I think it makes more sense to just use the `torch.cdist()` function across all kernel functions